modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,137 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any organization.
+Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization, including the Federated ones.
 
-In addition to Mule OAuth 2.0 Provider, MuleSoft offers out-of-the-box integration with PingFederate, OpenAM and Open ID Connect OAuth 2.0 Providers.
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
```

Comment:
I made this copyedit before as well. Why are you not making these changes?
```suggestion
You must deploy Mule OAuth 2.0 Provider to a runtime with API gateway capabilities, such as Mule runtime engine (Mule) 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third-party OAuth 2.0 providers are supported:
```

Reply:
When I opened the file to edit, it was already lower case. Strange! I now changed all instances to "Mule OAuth 2.o Provider".